### ISSUE TITLE
Bitcoin_Projection_Review

### ISSUE THREAD

**Issue by ShrutiSanghavi** [fjt37/ml-big-data-project#16]
The project started off with a great introduction of bitcoins and contained all the relevant information a user would need to understand the work done on the project. It further added to the excitement of reading the project, as in itself it sounds very interesting. The features were classified into two categories to help comprehended better. Understanding the reason for the price for be only flat in the first 3 years or so, (owing to Bitcoin not being traded publicly), and making a decision to drop the initial part of the dataset perhaps seems justified in this setting. It was interesting how to the group took into account the possibility of political events affecting the real value of the currency, and making an attempt to incorporate this feature into the data. The report is structured and explained well in theory and visuals, and care is taken to justify all decisions and inferences at all points in the project report. Considering it being such a hot topic, stating a disclaimer about the fickle success rate of the model was very thoughtful!  

I believe the team could use the following pointers to address some issues that I personally feel are not covered in the project. In the preliminary model analysis, in the second test, it is mentioned that the model overfit at 800 features, and about 70 of them were dropped. I, However, do not understand the meaning of the 800 features, since the original dataset contained only 24 features. Since only regression was carried out using different loss functions, I would have been excited to see some other techniques used for the project, including regression trees or some other form of clustering, although given the size of the working dataset is small, these could easily overfit. Maybe, some justification into why not considering any other methods except for regression should have been stated. I loved the project, and the thought of trying to model something that seems so relevant in this day, could be very powerful. I commend you all for your great work and success!
